sass/site/_layout.scss

```diff
@@ -1,15 +1,15 @@
 .corp-container {
 	margin: 0 auto;
-	max-width: none; 
+	max-width: none;
 	padding: 0 15px;
 	position: relative;
-	width: 1170px; // 1140px container.
-	
+	width: 1170px; // 1170px container.
```

Comment:
The initial comment is correct. The actual size given the box model is 1140.

Reply:
Ok. Making the change